cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/CassandraDialect.java

```diff
@@ -534,6 +534,12 @@ public DuplicateInsertPreventionStrategy getDuplicateInsertPreventionStrategy(En
 	}
 
 	@Override
+	public int executeBackendUpdateQuery(BackendQuery<String> query, QueryParameters queryParameters) {
```

Comment:
There is an abstract base class shared by some dialect implementations. `CassandraDialect` proably would benefit from extending it.